### PR TITLE
Add Equations 1.2.1+8.10 package, featuring HoTT compatibility

### PR DESCRIPTION
**Pull request by mattam82**

### CODE REVIEW

**Comment by mattam82** [comment]
@spitters, please test installing using `opam install coq-hott.8.10.dev coq-equations` . HoTT is now taken as an optional dependency: if it is installed then we build the HoTT-compatible version of Equations, otherwise the standard one.

**Comment by palmskog** [comment]
Any chance for a proper HoTT packaged release for `coq-released`? Even if it's only a depopt, this package makes the `coq-released` repo dependent on a git-based package in `coq-extra-dev` (which could theoretically fail whenever a commit is made).

File: released/packages/coq-equations/coq-equations.1.2.1+8.10/opam [hunk 1]
```diff
@@ -0,0 +1,40 @@
+opam-version: "2.0"
+authors: [ "Matthieu Sozeau <matthieu.sozeau@inria.fr>" "Cyprien Mangin <cyprien.mangin@m4x.org>" ]
+dev-repo: "git://github.com/mattam82/Coq-Equations.git"
```
Comment (palmskog) [inline released/packages/coq-equations/coq-equations.1.2.1+8.10/opam]:
```suggestion
dev-repo: "https://github.com/mattam82/Coq-Equations.git"
```

File: released/packages/coq-equations/coq-equations.1.2.1+8.10/opam [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+maintainer: "matthieu.sozeau@inria.fr"
+homepage: "https://mattam82.github.io/Coq-Equations"
+bug-reports: "https://github.com/mattam82/Coq-Equations/issues"
+license: "LGPL 2.1"
```
Comment (palmskog) [inline released/packages/coq-equations/coq-equations.1.2.1+8.10/opam]:
```suggestion
license: "LGPL-2.1-or-later"
```

File: released/packages/coq-equations/coq-equations.1.2.1+8.10/opam [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  [make "test-suite"]
+]
+depends: [
+  "coq" {>= "8.10~" & < "8.11~"}
+]
+depopts: [ "coq-hott" {= "8.10.dev" } ]
+url {
+  src:
+    "https://github.com/mattam82/Coq-Equations/archive/v1.2.1-8.10.tar.gz"
+  checksum: "md5=79ec88f048c233c311ab29dd732d2717"
```
Comment (palmskog) [inline released/packages/coq-equations/coq-equations.1.2.1+8.10/opam]:
```suggestion
  checksum: "sha512=b0c1771133e25afc915434535d7ff9ee77bb2c3f93b18cbb4d5f1b5270abad83a9c21d06218f47b271f94647fd38fbc688d0d2062f987164dbf7e1454e671a39"
```

**Comment by mattam82** [comment]
It will only build the HoTT version if hott.8.10.dev is installed itself, so I thought it would be a safe thing to do. A release of HoTT should come soon I guess.

**Pull request merged**